css/details.css

```diff
@@ -0,0 +1,154 @@
+body {
+    overflow-x: hidden;
+}
+
+.navbar-custom {
+    padding: 12px;
+    background-color: #F1F1F1;
+    margin-bottom: 0px;
+}
+
+.loklak-brand {
+    font-family: "Courier New";
+    font-size: 35px;
+    text-decoration: none !important;
+    line-height: 20px;
+    color: rgb(51,51,51);
+    line-height: 20px;
+}
+
+.loklak-header {
+    color: #C8254D;
+}
+
+.sidebar {
+    position: fixed;
+    width: 16%;
+    background-color: #F1F1F1;
+    height: 100%;
+    overflow-y: auto;
+    margin-top: 75px;
+}
+
+.sidebar-header {
+    background-color: #8BC34A;
+    color: white;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: start;
+    padding-left: 16px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+}
+
+.sidebar-header-image {
+    margin-right: 20px;
+}
+
+.menu-list {
+    margin-left: 0px;
+    padding-left: 0px;
+}
+
+.menu-list-item {
+    text-decoration: none !important;
```

Comment:
Could you prevent using !important as it leads to more errors in future. 
see this -> https://stackoverflow.com/questions/9245353/what-does-important-in-css-mean